lib/sshkit/backends/connection_pool.rb

```diff
@@ -1,5 +1,4 @@
 require "monitor"
-require "thread"
```

Comment:
🗒️ RuboCop flagged this as an unnecessary `require` (the `Thread` constant is always available without needing to require it.)